docker/python3-scientific/create_zarr_optimus.py

```diff
@@ -224,6 +225,40 @@ def add_cell_metrics(data_group, input_path, cell_ids, verbose=False):
         logging.info("Not adding \"cell_metadata_numeric\" to zarr output: either the #genes or # cell ids is 0")
 
 
+def create_gene_id_name_map(gtf_file):
+    """ Creates a map from gene_id to gene_name by reading in the GTF file
+
+    Args:
+        annotation_file (string): annotation file 
+
+    Return:
+        gene_id_name_map (dict[string]->string): dictonary gene ids to gene names
+    """
+    gene_id_name_map = {}
+
+    try:
+        if gtf_file.endswith(".gz"):
+            fpin = gzip.open(gtf_file, 'rt')
+        else:
+            fpin = open(gtf_file, 'r')
+    except FileNotFoundError:
+        print("GTF File not found")
```

Comment:
Do you really want to swallow the exception here? Seems like the function should fail at this point.

Reply:
I agree! removed the exception-handling part.